From 19eba34ac7cbf8af662fa8b01d8dcaf96c5f651c Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Mon, 21 Mar 2022 18:27:45 +0000
Subject: [PATCH] dtoverlays: Connect the backlight to the pitft35 display

DRM will automatically handle the backlight with the display if
told about the linking, so link the two together.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/pitft35-resistive-overlay.dts | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/overlays/pitft35-resistive-overlay.dts b/arch/arm/boot/dts/overlays/pitft35-resistive-overlay.dts
index 713532cb2272..e11e1920024a 100644
--- a/arch/arm/boot/dts/overlays/pitft35-resistive-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pitft35-resistive-overlay.dts
@@ -102,7 +102,7 @@ stmpe_gpio: stmpe_gpio {
 	fragment@5 {
 		target-path = "/soc";
 		__overlay__ {
-			backlight {
+			backlight: backlight {
 				compatible = "gpio-backlight";
 				gpios = <&stmpe_gpio 2 0>;
 				default-on;
@@ -115,6 +115,7 @@ __overrides__ {
 		rotate =  <&pitft>,"rotate:0";
 		fps =     <&pitft>,"fps:0";
 		debug =   <&pitft>,"debug:0";
-		drm =     <&pitft>,"compatible=adafruit,yx350hv15";
+		drm =     <&pitft>,"compatible=adafruit,yx350hv15",
+			  <&pitft>,"backlight:0=",<&backlight>;
 	};
 };
-- 
2.32.0

